src/component/Board.tsx

```diff
@@ -64,8 +64,8 @@ export const Board = (props: Props) => {
   );
 
   return (
-    <div className="board">
-      <div className="race">
+    <>
+      <>
```

Comment:
What are all these empty tags for?

Reply:
@jgayfer No reason apparently, they've been removed, thank you